.eslintrc.json

```diff
@@ -11,7 +11,8 @@
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "ecmaVersion": "latest",
-    "sourceType": "module"
+    "sourceType": "module",
+    "project": ["./tsconfig.base.json"]
```

Comment:
Setting the `@typescript-eslint/no-floating-promises` rule causes this error: `You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.`, so specify a project.